proposed_changes.txt

```diff
@@ -0,0 +1,81 @@
+Some ideas on modifying data representation.
+This commit is not intended to be merged. It is only intended for discussion.
+If it is decided that some changes make sense, then they shall be handled by multiple separate commits or pull requests.
+
+Idea 1:
+
+Use float (or in rare cases double) instead of int-based datatypes for all datapoints where fractional numbers might be relevant.
+(for datapoints like Axle.WheelCount it is natural to use an int-based type)
+
+Idea 2: 
+
+Stick to the same representation for all datapoints using a particular unit, e.g. always use float for km/h.
+That could simplify implementation of handlers for that particular unit.
+(This could long term mean that there is no need to specify both unit and datatype, one of them should be sufficient)
+
+Shown example changes in this commit (for idea 1 and/or 2):
+-Always use float for: cm, l, km/h, celsius, inch, degrees/s
+-Always use uint16 for mm 
+-Always use uint32 for s 
+-Always use int32 for rpm (signed to allow support for electrical engines)
+
+Idea 3:
+
+Consider removing size-specific datatypes (e.g. int16, uint32)
+By just supporting e.g. int (and representing it as e.g. int32) or float/double we will avoid lengthy discussions 
+on hypothetical supported range for a datapoint. Most likely we can represent all integer datapoints with int32.
+Size limits can anyway be specified with min/max, if needed.
+
+Idea 4: 
+
+Remove limits in standard, unless there is a standard/norm enforcing those limits, or the limits are natural (e.g. percent, degrees).
+If limit set by standard/norm, include a rationale for the limit.
+Reason: Limits are likely OEM/Vehicle-specific anyway, should possibly be specified in a similar way that
+        attributes are set 
+        (e.g. when you declare vehicle attributes like length and weight you can also declare supported range for the climate control)
```

Comment:
That was discussed already, and wasn't in scope vor VSS2. My opinion is "min" "max" and similar things are deployment specific. This was already touched here: https://github.com/GENIVI/vehicle_signal_specification/issues/166